src/app/container/container.component.html

```diff
@@ -13,7 +13,8 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-<div *ngIf="tool || !displayAppTool; else appTool">
+<app-workflow *ngIf="displayAppTool" [isWorkflowPublic]="isToolPublic"></app-workflow>
+<div *ngIf="tool && !displayAppTool">
```

Comment:
The previous logic that controlled whether to use the tool template (for old-style tools) or the workflow template) for apptools) was incorrect.  It's fixed here, and I moved the code that conditionally delelegates to the workflow template to the top, to make sure it's clear what's happening without having to scan the entire template.